### PR TITLE
Display word numbers in seed phrase

### LINKED ISSUE #2961: Numbers with seed phrase
When you generate the seed phrase, the words should have a number first so you can keep track of which words are which. Copy and pasting this can be confusing and words can be jumbled.

Add the appropriate number in front of each seed word.

For new users I have seen that sometimes when copy/paste the words have problems to recover the wallet, as a help to new users could be useful.
Thanks for all!!

### PR DESCRIPTION
**Pull request by mariopino**
Closes #2961

**Description:**

Create a component to display a 24 words seed phrase showing the word number. 

image

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by faboweb** [comment]
Sweet! :)

**Comment by mariopino** [comment]
Added warning:

image

Would be great that warning fits in a single line to avoid horizontal scrolling for viewports height under 700px.

Suggestions / feedback welcome!

File: vue.config.js [hunk 1]
```diff
@@ -15,6 +15,9 @@ module.exports = {
   publicPath: `/`,
   configureWebpack: () => {
     const config = {
+      devServer: {
```
Comment (faboweb) [inline vue.config.js]: this should go into another PR as it probably breaks things

Comment (mariopino) [reply vue.config.js]: Agree! Was included by error because I use it for testing.
